pandas/core/indexes/base.py

```diff
@@ -2788,7 +2789,7 @@ def get_loc(self, key, method=None, tolerance=None):
             positions matches the corresponding target values. Missing values
             in the target are marked by -1.
 
-        Examples
+        %(raises_section)sExamples
```

Comment:
should this not be on a separate line? (maybe remove the blank lines at the end of the template)

Reply:
Thanks @jreback that does look a lot cleaner - I've adjusted the substitution so its on its own line.